.github/workflows/tests.yml

```diff
@@ -15,10 +15,10 @@ jobs:
       - uses: actions/checkout@v1
       - name: Install pre-reqs
         run: |
-          sudo apt-get install -y yamllint
-          sudo add-apt-repository ppa:rmescandon/yq
           sudo apt update
-          sudo apt install yq -y
+          sudo apt-get install -y yamllint
+          sudo apt install snapd
+          sudo snap install yq
```

Comment:
do not use snap to install yq the performance is horrible in azure devops

https://hmcts-reform.slack.com/archives/C01APTJAM7D/p1616057990016500